setup.py

```diff
@@ -66,7 +45,7 @@ def read(filename):
         'Programming Language :: Python :: 3.8',
     ],
     python_requires='>=3.6',
-    install_requires=['setuptools'],
```

Comment:
if I read the [documentation](https://setuptools.readthedocs.io/en/latest/setuptools.html) on `setup_requires` correctly, we need `install_requires` **and** `setup_requires`: `pint` depends on `pkg_resources`.